js/index.js

```diff
@@ -1 +1,105 @@
 // Your code goes here
+const navItem = Array.from(document.querySelector('nav').children);
+const hide = document.querySelectorAll('.hide');
+const imgList = Array.from(document.querySelectorAll('img'));
+const submit = document.querySelector('#submit');
+const overlay = document.querySelector('.overlay');
+const button = Array.from(document.querySelectorAll('.btn'));
+const cancel = document.querySelector('#cancel');
+const form = document.querySelector('form');
+const heading = document.querySelector('h1');
+const footer = document.querySelector('footer');
+///1. click event listener
+//Navigation 'active' functionality.
+for (let i=0; i < navItem.length; i++) {
+
+  let activeIcon = navItem[i].nextElementSibling; //selects active icon
+  navItem[i].addEventListener('click',(e)=> { //adds click listener to nav links
+        e.preventDefault();
+    for (let i=0; i < hide.length; i ++) { //loops through all icons and turns their active stylings 'off'
+      hide[i].className = 'hide';
+      navItem[i].className = 'navItem';
+    }
```

Comment:
You can also just use `querySelector` to find the active nav link icon without looping.